src/handle-completed.js

```diff
@@ -0,0 +1,15 @@
+const tasks = JSON.parse(localStorage.getItem('arrayOfTasks')) || [];
+
+export const handleComplected = (event, task) => {
+  if (!task.completed) {
+    task.completed = true;
+  } else {
+    task.completed = false;
+  }
+  return task;
+};
+
+export const removeCompetedTasks = (tasks) => {
+  const notCompletedTasks = tasks.filter((task) => !task.completed);
+  return notCompletedTasks;
+};
```

Comment:
:red_circle: [REQUIRED] I noticed that whenever you reload or create a new task, the changes made to the completed state are lost. This is because the local storage state is not being updated when the completed state is updated. Please make sure to update the local storage state whenever the completed state is updated.

![share 2](https://github.com/majdalkilany/To-Do-List/assets/26736582/ca97e30d-ed0c-4c47-9d89-5cef4679ca82)